packages/poaps/src/PoapsClient.ts

```diff
@@ -137,7 +134,25 @@ export class PoapsClient {
    * @returns {Promise<GetMintCodeResponse>} The mint code details.
    */
   async getMintCode(mintCode: string): Promise<GetMintCodeResponse> {
-    return await this.tokensApiProvider.getMintCode(mintCode);
+    const getMintCodeRaw = await this.tokensApiProvider.getMintCode(mintCode);
+    return {
+      id: getMintCodeRaw.id,
+      qrHash: getMintCodeRaw.qr_hash,
+      txHash: getMintCodeRaw.tx_hash,
+      eventId: getMintCodeRaw.event_id,
+      beneficiary: getMintCodeRaw.beneficiary,
+      userInput: getMintCodeRaw.user_input,
+      signer: getMintCodeRaw.signer,
+      claimed: getMintCodeRaw.claimed,
+      claimedDate: getMintCodeRaw.claimed_date,
+      createdDate: getMintCodeRaw.created_date,
+      isActive: getMintCodeRaw.is_active,
+      secret: getMintCodeRaw.secret,
+      txStatus: getMintCodeRaw.tx_status,
+      result: {
+        token: getMintCodeRaw.result.token,
+      },
+    };
```

Comment:
This object doesn't seem to be normalized. It says `claimed` instead of `minted` or `reserved` depending if it was an email or an address.